math-scala/src/test/scala/org/apache/mahout/math/algorithms/RegressionTestsSuiteBase.scala

```diff
@@ -106,4 +106,83 @@ trait RegressionTestsSuiteBase extends DistributedMahoutSuite with Matchers {
     (myAnswer - correctAnswer) should be < epsilon
   }
 
+  test("OLS w/Ftest and Ttest validation") {
+
```

Comment:
actually this all can be moved to the "fitness tests" , use the R-prototype in there and just confirm the F-test, should be in good shape.